app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :likes, foreign_key: :user_id
+  has_many :comments, foreign_key: :user_id
+end
```

Comment:
- [ ] Good job working on this but you did not include a method that returns the three most recent post that belongs to a user.